latest/llms.txt

```diff
@@ -0,0 +1,29 @@
+# NVIDIA CUDA-Q
+
+> CUDA-Q streamlines hybrid application development and promotes productivity and scalability in quantum computing. It offers a unified programming model designed for a hybrid setting—that is, CPUs, GPUs, and QPUs working together. CUDA-Q contains support for programming in Python and in C++.
```

Comment:
What do you think about using the word "heterogeneous" instead of "hybrid" here?